src/azul/pagination.py

```diff
@@ -0,0 +1,35 @@
+from azul.requests import parse_url_qs
+
+internal_pagination_parameters = ('search_after', 'search_after_uid', 'search_before', 'search_before_uid')
+'''
+    These pagination parameters are used internally, but should removed from the response that a client receives.
+'''
+
+
+def extract_internal_pagination_params_from_url(url: str):
+    """
+       Helper method that extracts parameters for pagination from a url
+    """
+    query_params = parse_url_qs(url)
+    found_params = dict()
+    for x in internal_pagination_parameters:
+        found_params[x] = query_params.get(x)
+    return found_params
+
+
+def extract_internal_pagination_params(json_response):
+    """
+       Helper method that extracts parameters for pagination from a response
+    """
+    next_url = json_response['pagination'].get('next')
+    previous_url = json_response['pagination'].get('previous')
+    joint_url_params = {x: None for x in internal_pagination_parameters}
```

Comment:
Don't use None values. Absence and use of .get() is preferred.

Reply:
`None` is used here to allow for easier merging of the extracted parameter values.
The code will become more complex if there is a hard requirement for its removal.